.github/workflows/helm.yaml

```
@@ -34,12 +34,12 @@ jobs:
         run: ct lint --config ct.yaml
 
       - name: Create kind cluster
-        uses: helm/kind-action@a1b0e391336a6ee6713a0583f8c6240d70863de3 # v1.12.0
+        uses: helm/kind-action@v1.12.0
         if: steps.list-changed.outputs.changed == 'true'
         with:
-          version: v0.23.0
-          node_image: kindest/node:v1.32.0
-          kubectl_version: v1.32.0
+          version: v0.24.0
```

This kind version should match the version specified in aqua.yaml.

```suggestion
          version: v0.29.0
```